screen-wake-lock/wakelock-screen-type-on-worker.https.worker.js

```diff
@@ -1,8 +0,0 @@
-//META: title=Screen wake lock should not be allowed in dedicated worker
-importScripts("/resources/testharness.js");
-
-promise_test(t => {
-  return promise_rejects_dom(t, "NotAllowedError", navigator.wakeLock.request('screen'));
-}, "Screen wake lock should not be allowed in dedicated worker");
```

Comment:
Why remove this test?

Reply:
The test was simply wrong. There is nothing in the spec that exposes the API on the `WorkerNavigator` interface in workers, so running the above would just fail on `navigator.wakeLock` because `.wakeLock` is would be `undefined`.